viewing_party/party.py

```diff
@@ -1,23 +1,123 @@
 # ------------- WAVE 1 --------------------
 
+from hashlib import new
+from re import M
+from collections import Counter
+from viewing_party.helper_functions import *
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1, USER_DATA_3
+
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        new_movie = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return new_movie
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched": [movie]
+    }             
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist": [movie]
+    }             

```

Comment:
Use the append method here instead so the existing "watchlist" doesn't get replaced by `[movie]`